.github/workflows/src/context.js

```diff
@@ -246,26 +246,42 @@ export async function extractInputs(github, context, core) {
         `Could not extract build ID or project URL from check run details URL: ${checkRun.details_url}`,
       );
     }
-    if (
-      !context.payload.repository ||
-      !context.payload.repository.owner ||
-      !context.payload.repository.owner.login ||
-      !context.payload.repository.name
-    ) {
-      throw new Error(
-        `Could not extract repository owner or name from context payload: ${JSON.stringify(context.payload.repository)}`,
+
+    const payload =
+      /** @type {import("@octokit/webhooks-types").CheckRunEvent} */ (
+        context.payload
       );
-    }
+
     inputs = {
-      owner: context.payload.repository.owner.login,
-      repo: context.payload.repository.name,
+      ...getRepositoryInfo(payload.repository),
```

Comment:
```suggestion
      owner: repositoryInfo.owner,
      repo: repositoryInfo.repo,
```

I prefer being explicit when possible, rather than using the spread operator, since I find this more readable.